extension/src/embed_scripts/GoogleClassroomOverlay.js

```diff
@@ -119,6 +124,105 @@ function addYoutubeIFrame(rawDestination) {
     });
 }
 
+///////////////////////////////////////////
+/////////// Webcam feed logic 
+///////////////////////////////////////////
+let video, canvas, ctx, model, stream;
+
+async function settingUpModel() {
+    await tf.wasm.setWasmPath("https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-backend-wasm/dist/tfjs-backend-wasm.wasm"); // eslint-disable-line no-undef
```

Comment:
in the `eslintrc` file, let's add `tf` and `blazeface`. Using inline exception is fine but it should not be abused